central/processlisteningonport/store/postgres/index.go

```diff
@@ -0,0 +1,70 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+package postgres
+
+import (
+	"context"
+	"time"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	metrics "github.com/stackrox/rox/central/metrics"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	storage "github.com/stackrox/rox/generated/storage"
+	ops "github.com/stackrox/rox/pkg/metrics"
+	search "github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/blevesearch"
+	"github.com/stackrox/rox/pkg/search/postgres"
+	"github.com/stackrox/rox/pkg/search/postgres/mapping"
+)
+
+func init() {
```

Comment:
#4350 moves this to `pkg/postgres/schema`.  So watch for when that one is merged and adjust accordingly.

Reply:
Already done, it was preventing the pipeline runs.